test/integration/addons_knative_test.go

```diff
@@ -12,9 +12,11 @@ import (
 	"github.com/kong/kubernetes-testing-framework/pkg/clusters/addons/knative"
 )
 
+const knativeVersion = "knative-v1.10.0"
```

Comment:
This is OK-ish but feels a bit weird to store a "prefix-version" in a `const` which is named `knativeVersion`. I think I'd prefer to have just the semver in this `const` and use the `knative-` when calling `WithVersion()` but I'm not going to push it.